src/state/application/updater.ts

```diff
@@ -42,6 +48,27 @@ export default function Updater(): null {
     [chainId, setState]
   )
 
+  const chainConnectivityWarningActive = useAppSelector((state) => state.application.chainconnectivityWarning)
+  const timeout = useRef<NodeJS.Timeout>()
+  useEffect(() => {
+    const waitMSBeforeWarning =
+      (chainId ? CHAIN_INFO[chainId]?.blockWaitMSBeforeWarning : DEFAULT_SECONDS_BEFORE_WARNING_WAIT) ||
```

Comment:
may be worth converting everything to use ms or s? I think you'd need to multiple `DEFAULT_SECONDS_BEFORE_WARNING_WAIT` by `1000` here?